.circleci/config.yml

```diff
@@ -96,8 +95,7 @@ jobs:
         - run:
             name: Get data
             command: |
-              python setup.py develop
-              mkdir -p ~/mne_data
+              pip install -e .[docs]
```

Comment:
one disadvantage of doing this here rather than above is that the `pip cache` step won't cache the right stuff anymore. I think it's okay to move this install line above